src/components/v5/shared/Button/Button.tsx

```diff
@@ -28,6 +28,8 @@ const Button: FC<PropsWithChildren<ButtonProps>> = ({
   iconName,
   iconSize = 'tiny',
   isIconRight,
+  onTrackClick,
```

Comment:
I think `onTrackClick` is not necessary, we can just track the event in the button's own `onClick` handler